packages/eslint-plugin-office-addins/src/rules/no-empty-load.ts

```diff
@@ -0,0 +1,34 @@
+import { TSESTree } from "@typescript-eslint/experimental-utils";
+
+export = {
+  name: "no-empty-load",
+  meta: {
+    type: <"problem" | "suggestion" | "layout">"problem",
+    messages: {
+      emptyLoad:
+        "Calling an empty load can slow down your add-in",
```

Comment:
"calling an empty load" might be better explained by saying "Calling load without any argument".